compat/mingw.c

```diff
@@ -661,8 +661,18 @@ static int mingw_open_append(wchar_t const *wfilename, int oflags, ...)
 	handle = CreateFileW(wfilename, FILE_APPEND_DATA,
 			FILE_SHARE_WRITE | FILE_SHARE_READ,
 			NULL, create, FILE_ATTRIBUTE_NORMAL, NULL);
-	if (handle == INVALID_HANDLE_VALUE)
-		return errno = err_win_to_posix(GetLastError()), -1;
+      
+        if (handle == INVALID_HANDLE_VALUE) {
+        
+	        DWORD err = GetLastError();
+
+		// Some network storage solutions (e.g. Isilon) might return ERROR_INVALID_PARAMETER 
```

Comment:
In Git's source code, there are only `/* ... */` style comments, not C++/C99 ones. Please change this comment accordingly.

Reply:
I indeed should have used the same style that was already used in the existing code